### PR TITLE
Audit kyuubi operation state change log

### PR DESCRIPTION
**Pull request by turboFei**
# :mag: Description
## Issue References 🔗

We meet some issue, I want to check the operation sessionHandle and the operation timeline.
I found that, it is difficult for me to check it from kyuubi log.
So, In this pr, I log the operation change log into separate file.
## Describe Your Solution 🔧

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Types of changes :bookmark:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Related Unit Tests

---

# Checklist 📝

- [x] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

**Comment by turboFei** [comment]
cc @pan3793

File: kyuubi-server/src/test/resources/log4j2-test.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
 <Configuration status="WARN">
```
Comment (pan3793) [inline kyuubi-server/src/test/resources/log4j2-test.xml:21]: we have bunches of `log4j2-test.xml`, can we sync them?

**Review by cxzl25 — APPROVED** [review]

File: kyuubi-server/src/test/resources/log4j2-test.xml (hunk 1 above)
Comment (turboFei) [reply kyuubi-server/src/test/resources/log4j2-test.xml:21]: I think it is not needed, we just need to enable it in kyuubi-server side.

**Comment by pan3793** [comment]
Thanks, merged to master

**Pull request closed without merging**